docs/en/query_language/functions/string_functions.md

```diff
@@ -115,13 +115,104 @@ SELECT format('{} {}', 'Hello', 'World')
 └───────────────────────────────────┘
 ```
 
-## concat(s1, s2, ...)
+## concat {#concat}
 
 Concatenates the strings listed in the arguments, without a separator.
 
-## concatAssumeInjective(s1, s2, ...)
+**Syntax** 
 
-Same as [concat](./string_functions.md#concat-s1-s2), the difference is that you need to ensure that concat(s1, s2, s3) -> s4 is injective, it will be used for optimization of GROUP BY
+```sql
+concat(s1, s2, ...)
+```
+Alias: `CONCAT`.
```

Comment:
It's just case-insensitive:
```
SELECT cOncat('aa', 'bb')
FORMAT TSV

aabb
```

While the real alias is `||` binary operator which is part of SQL standard:
```
:) select 'aa' || 'bb' format TSV;

SELECT concat('aa', 'bb')
FORMAT TSV

aabb
```